control/joystick_commander/src/commander.c

```diff
@@ -50,144 +50,97 @@
 // *****************************************************
 
 /**
- * @brief Throttle axis index.
+ * @brief Joystick axis indices
  *
  */
 #define JOYSTICK_AXIS_THROTTLE (5)
-
-
-/**
- * @brief Brake axis index.
- *
- */
 #define JOYSTICK_AXIS_BRAKE (2)
-
-
-/**
- * @brief Steering axis index.
- *
- */
 #define JOYSTICK_AXIS_STEER (3)
 
 
 /**
- * @brief Enable controls button index.
+ * @brief Joystick button indices
  *
  */
 #define JOYSTICK_BUTTON_ENABLE_CONTROLS (7)
-
-/**
- * @brief Disable controls button index.
- *
- */
 #define JOYSTICK_BUTTON_DISABLE_CONTROLS (6)
 
 /**
- * @brief Maximum allowed throttle pedal position value. [normalized]
+ * @brief Throttle pedal position values [normalized]
  *
  */
+#define MIN_THROTTLE_PEDAL (0.0)
```

Comment:
Yes, I changed these so that the constant used to check the range of the throttle and brake pedals was not a "magic number."  If we choose to change the range in the future, these definitions are the only ones that need to be changed.